website/src/queries-logic.js

```diff
@@ -237,6 +229,119 @@ function add_fork_elements(forkdata_array, user, repo, parentDefaultBranch) {
   }
 }
 
+function update_filter() {
+  let is_useful_fork = getFilterFunction()
+  if (typeof is_useful_fork === 'function') {
+    update_table(TABLE_DATA.filter(is_useful_fork));
+  } else {
+    update_table(TABLE_DATA);
+  }
```

Comment:
Seems like this is repeated twice. Maybe could be extracted as a method?

Reply:
I used to try and my idea about the name is `update_table_trying_use_filter`. But I finally abort since it doesn't "deserve".

Certainly, it's also reasonable to extract that as a method because its meaning is also clear. If you stick.

Reply:
`update_table_trying_use_filter` sounds good enough to me. Feel free to update with it.